tests/test_fabricpath_topology.rb

```diff
@@ -0,0 +1,98 @@
+# Copyright (c) 2014-2015 Cisco and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require_relative 'ciscotest'
+require_relative '../lib/cisco_node_utils/cisco_cmn_utils'
+require_relative '../lib/cisco_node_utils/fabricpath_topology'
+
+include Cisco
+
+# TestFabricpathTopo - Minitest for Fabricpath Topo node utils
+class TestFabricpathTopo < CiscoTestCase
+  def setup
+    # setup runs at the beginning of each test
+    super
+    skip("Test not supported on #{node.product_id}") if
+      cmd_ref.lookup('fabricpath', 'feature').default_value.nil?
+    no_feature_fabricpath
+  end
+
+  def teardown
+    # teardown runs at the end of each test
+    no_feature_fabricpath
+    super
+  end
+
+  def no_feature_fabricpath
+    # Turn the feature off for a clean test.
+    config('no feature-set fabricpath')
+    config('no install feature-set fabricpath')
```

Comment:
Do you actually need to do a 'no install' every time? It's somewhat time-consuming to do it for every test case. I would think the 'no feature-set' by itself would be sufficient.